### PR TITLE
Remove output in class and create decent message

### PR DESCRIPTION
**Pull request by ed-holland**
Issue #74

### CODE REVIEW

File: src/deps/zcl_abaplint_deps_find.clas.abap [hunk 1]
```diff
@@ -6,30 +6,23 @@ CLASS zcl_abaplint_deps_find DEFINITION
...
+        !iv_max_level TYPE i DEFAULT 20 .
     METHODS find_by_item
       IMPORTING
         !iv_object_type TYPE trobjtype
         !iv_object_name TYPE sobj_name
       RETURNING
         VALUE(rt_tadir) TYPE zif_abapgit_definitions=>ty_tadir_tt
       RAISING
-        zcx_abapgit_exception .
-    METHODS find_by_item_minimal
```
Comment (larshp) [inline src/deps/zcl_abaplint_deps_find.clas.abap:19]: I added this method yesterday

File: src/deps/zcl_abaplint_deps_find.clas.abap [hunk 2]
```diff
@@ -40,7 +33,7 @@ CLASS zcl_abaplint_deps_find DEFINITION
         devclass     TYPE devclass,
       END OF ty_tadir .
     TYPES:
-      ty_tadir_tt TYPE SORTED TABLE OF ty_tadir WITH UNIQUE KEY ref_obj_type ref_obj_name.
```
Comment (larshp) [inline src/deps/zcl_abaplint_deps_find.clas.abap:43]: plus changed this to a sorted table for performance

File: src/deps/zcl_abaplint_deps_find.clas.abap [hunk 3]
```diff
@@ -154,48 +140,43 @@ CLASS ZCL_ABAPLINT_DEPS_FIND IMPLEMENTATION.
...
     LOOP AT it_senvi INTO ls_senvi.
       "Translate when required
       IF ls_senvi-type = 'BADI'. "Ignore
         CONTINUE.
 
       ELSEIF ls_senvi-type = 'INCL'. "Include is PROG
-        CLEAR ls_tadir.
-        ls_tadir-ref_obj_type = 'PROG'.
-        ls_tadir-ref_obj_name = ls_senvi-object.
-        INSERT ls_tadir INTO TABLE rt_tadir.
```
Comment (larshp) [inline src/deps/zcl_abaplint_deps_find.clas.abap:170]: and changed all of these to work with sorted tables

File: src/deps/zcl_abaplint_deps_find.clas.abap [hunk 4]
```diff
@@ -496,12 +417,6 @@ CLASS ZCL_ABAPLINT_DEPS_FIND IMPLEMENTATION.
           iv_level = iv_level
         CHANGING
           ct_tadir = lt_tadir ).
-    ELSEIF is_object-object = 'DTEL'.
-      find_dtel_dependencies(
```
Comment (larshp) [inline src/deps/zcl_abaplint_deps_find.clas.abap:500]: and added special handling for data elements

**Comment by ed-holland** [comment]
> suggest starting over with this PR, it removes stuff from master

I've noticed that also and added them again yesterday, but other merges did not get in either... will restart

**Pull request closed without merging**